product_variant_route_mto/__manifest__.py

```diff
@@ -9,10 +10,10 @@
     "category": "Inventory",
     "website": "https://github.com/OCA/product-attribute",
     "author": "Camptocamp SA, Odoo Community Association (OCA)",
-    "maintainers": ["mmequignon"],
+    "maintainers": ["mmequignon", "jbaudoux"],
     "license": "AGPL-3",
     "installable": True,
     "auto_install": False,
-    "depends": ["stock"],
+    "depends": ["product_route_mto"],
```

Comment:
Nitpick: your commit message should be `rely on product_route_mto` (which rely itself on `stock_route_mto`)